glossary.html

```diff
@@ -35,8 +36,8 @@ <h2>HTTP</h2>
 </main>
 <footer>
 	<hr>
-	<blockquote>"If you want to be happy, do not dwell in the past, do not worry about the future, focus on living fully in the present."</blockquote>
-	<blockquote> <span class="" value="byline">― Roy T. Bennett, The Light in the Heart<span></blockquote>
```

Comment:
use `class="byline"` instead of 'class="" value="byline"`.  The value in the quotation marks IS the value for an attribute like `class`